mlem/contrib/kubernetes/context.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+from typing import ClassVar, Optional
+
+from pydantic import BaseModel
+
+from mlem.utils.templates import TemplateModel
+
+logger = logging.getLogger(__name__)
+
+
+class K8sYamlBuildArgs(BaseModel):
+    name: Optional[str] = "mlem-app"
```

Comment:
Are they really optional? Meaning can they value be `None`? 
They are not required since you provided default value, but it's not the same as `Optional`

Reply:
yeah, on a 2nd thought, all are required I guess, removed `Optional`, thanks!